app/mails/views.py

```diff
@@ -147,7 +147,10 @@ class MailView(generic.ListView):
 
     @receiver(request_started)
     def import_mail(**kwargs):
-        management.call_command("import", verbosity=1)
+        try:
+            management.call_command("import", verbosity=1)
+        except Exception as exc:
+            print(exc)
```

Comment:
Good work.. But for future development, you will need to revisit this and use proper logging. OK for now though.